admin/app/tools/charts/charts.scala

```diff
@@ -128,11 +128,12 @@ class AwsLineChart(
       .getOrElse(throw new IllegalStateException(s"Don't know how to get a value for $dataPoint"))
 
   protected def toLabel(date: DateTime): String = date.withZone(format.timezone).toString("HH:mm")
+
+  lazy val latest = dataset.lastOption.flatMap(_.values.headOption).getOrElse(0.0)
```

Comment:
Moving `latest` so it is also available for `AwsLineChart` and not just for `AwsDailyLineChart`